CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ option(ZLIB_BUILD_MINIZIP "Enable building libminizip contrib library" OFF)
 option(ZLIB_INSTALL "Enable installation of zlib" ON)
 option(ZLIB_PREFIX "prefix for all types and library functions, see zconf.h.in"
        OFF)
+option(DFLTCC "Enable building S390-DFLTCC implementaiton" OFF)
```

Comment:
ZLIB_DFLTCC
embeded into if((LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "s390x")

same for CRC32VX